components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -215,12 +215,18 @@ input OpenStackProviderConfigInput {
     loadBalancerProvider: String! # Name of load balancer provider, e.g. f5
 }
 
+enum KymaInstallationMethod {
+    KymaOperator
+    ParallelInstall
+}
+
 input KymaConfigInput {
     version: String!                            # Kyma version to install on the cluster
     profile: KymaProfile                        # Optional resources profile
+    kymaInstaller: KymaInstallationMethod       # Optional installation method (KymaOperator is default)
```

Comment:
We should also add kymaInstaller field to KymaConfig type to be able to return it in `runtimeStatus` query.